### PR TITLE
update to Swift 5.4.2 toolchain

### PR DESCRIPTION
**Pull request by michaelknoch**

### CODE REVIEW

**Comment by MihaelIsaev** [comment]
Can't wait for it! @michaelknoch thanks for your work!

File: setup.sh [hunk 1]
```diff
@@ -2,73 +2,37 @@
...
-    BASEPATH="https://swift-toolchain-artifacts.flowkeycdn.com"
+    log "Downloading Toolchain..."
 
-    if [[ ${UNAME} == "Darwin" ]]; then
-        VERSION="20200112.01"
-    elif [[ ${UNAME} == "Linux" ]]; then
-        VERSION="stable_2"
-    fi
+    # mirror of https://github.com/vgorloff/swift-everywhere-toolchain/releases/download/1.0.68/swift-android-toolchain.tar.gz
+    TOOLCHAIN_PATH="https://swift-toolchain-artifacts.flowkeycdn.com/swift-android-toolchain-5.3.3.tar.gz"
```
Comment (michaelknoch) [inline setup.sh:27]: our toolchain is just a thin wrapper around the great work of vgorloff. we should add some reference to the readme

**Comment by ephemer** [comment]
@michaelknoch are we upgrading to 5.4 or 5.3? Is the revert back to 5.3 because of crashes we saw in 5.4? It would be great if you could add credit to vgorloff in the readme 👍  Otherwise feel free to merge when you're ready

File: setup.sh (hunk 1 above)
Comment (ephemer) [reply setup.sh:27]: is this mac only? i think we should make that clear in the readme too if so

**Review by ephemer — APPROVED** [review]
LGTM but we should update the readme on a few points

**Comment by michaelknoch** [comment]
exactly @ephemer , the arm32 crashes were the reason to downgrade back to 5.3. https://github.com/vgorloff/swift-everywhere-toolchain/issues/123

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # swift-android-toolchain
 
-Build Swift for Android from Mac and Linux
+Build Swift for Android from Mac
```
Comment (michaelknoch) [inline README.md:3]: you think this isn't clear enough? @ephemer

File: setup.sh (hunk 1 above)
Comment (michaelknoch) [reply setup.sh:27]: seehttps://github.com/flowkey/swift-android-toolchain/pull/12/files#r665360193

File: README.md (hunk 2 above)
Comment (ephemer) [reply README.md:3]: Didn't see this sorry, yeah that's enough for the macOS only change!

**Pull request merged**